website/docs/brain/custom-applications.mdx

```diff
@@ -8,70 +8,94 @@ title: Developing Custom Applications
 :::tip
 We now have a full tutorial that walks you through creating custom applications that interact with cameras and drive your Amiga from the brain!
 
-Check it out at [Virtual Joystick Tutorial](./../examples/virtual_joystick/00_overview.md).
+Check it out at [Virtual Joystick Tutorial](/examples/virtual_joystick/00_overview.md).
```

Comment:
to avoid having relative paths: https://docusaurus.io/docs/markdown-features/links

i read also somewhere that we can specify by the markdown id but couldn't find where now

Reply:
Nice, that's much better